### PR TITLE
Fix path issue for highcharts-more & highcharts-3d

### PR DESCRIPTION
**Pull request by denchen**
In relation to https://github.com/ahmadsoe/ember-highcharts/issues/58 , fix issue with modules not loading for highcharts-more and highcharts-3d

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,29 +16,30 @@ module.exports = {
...
     if (options.includeHighCharts3D) {
-      app.import(highchartsPath + '/highcharts/highcharts-3d.src.js');
+      app.import(path.join(highchartsPath, 'highcharts-3d.src.js'));
     }
 
     if (options.includeModules) {
       var modules = options.includeModules;
       for (var i = 0; i < modules.length; i++) {
-        app.import(highchartsPath + '/modules/' + modules[i] + '.src.js');
+        var module = modules[i] + '.src.js';
```
Comment (gregjopa) [inline index.js]: Can you rename the variable `module` to `highchartsModule`? I imagine `module` will eventually become a reserved word in Node.js if they implement es6 modules.

**Comment by gregjopa** [comment]
Can you please squash the two commits into one?

To do so run `git rebase --interactive HEAD~2` and then pick the first commit.

**Comment by denchen** [comment]
I hope I did it correctly. I've never rebased before. What I did was squash the second commit into the first and did a force push.

**Comment by gregjopa** [comment]
@denchen looks good to me. Thanks for contributing!

**Comment by gregjopa** [comment]
@ahmadsoe can you please release a new version of this addon? This was a bugfix so I think we should bump the version to 0.4.1.

**Pull request merged**